proposals/NNNN-GetSystemTime.md

```diff
@@ -0,0 +1,63 @@
+# GetSystemTime RPC
+
+* Proposal: [SDL-NNNN](NNNN-GetSystemTime.md)
+* Author: [Aleksandr Stasiuk](https://github.com/AStasiuk)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC GetSystemTime to SDL Core for get real system time during security TLS handshake.

```

Comment:
Suggested text: 
This proposal is about adding a new RPC GetSystemTime inorder for SDL Core to get accurate system time information which is utilized by the TLS handshake mechanism.

Reply:
@robinmk Fixed in https://github.com/smartdevicelink/sdl_evolution/pull/324/commits/19088cdb433d915b8fe826acfcf622b427a02f1e